examples/encfs/TROUBLESHOOTING.md

```diff
@@ -78,6 +110,14 @@ Ensure that:
 
 - the url of the uploaded blob is correctly copied into the [`encfs-sidecar-args.json`](encfs-sidecar-args.json#L5) file and before base64 encoding the contents of the file and copying to the [`EncfsSideCarArgs`](aci-arm-template.json#L36) field in the ARM template
 
+- the managed identity on the ARM template has the correct role access to the storage account/container. 
+
+## Cannot lukesopen Error
+
+When checking the log output the ENCFS container, you may see the error that says "Cannot luksopen". 
+
+This happens when the released key cannot be used to decrypt remote file system. One possible cause is that the oct key was imported as an rsa key. Thus, re-examine the key you imported and re-import the key with the right `kty` indication. 
```

Comment:
I would just make a bullet here to tell them to configure the right `kty` on `importkeyconfig.json` to be consistent with style:
-  configure the right `kty` on `importkeyconfig.json`: currently the import key tool only supports `RSA-HSM` and `oct-HSM`